render.yaml

```diff
@@ -54,7 +54,7 @@ services:
 - type: web
   name: zulip
   env: docker
-  plan: standard plus
+  plan: starter plus
```

Comment:
Why are we changing the plan?

Reply:
We're running it in multithreaded mode (by setting `QUEUE_WORKERS_MULTIPROCESS` to `False` explicitly) instead of multiprocess mode, so it consumes a lot less memory.

[Docs](https://zulip.readthedocs.io/en/4.2/production/deployment.html#queue-workers-multiprocess) on Zulip

Reply:
👍 Sorry, I just noticed you explained that in the PR description. Thanks for double-explaining!